platform/persistence/platform-persistence-core-listeners/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD description="System Alert Listener"
+         name="Alert Listener" id="org.codice.ddf.persistence.events.AlertListener">
+
+        <AD description="The periodic interval on which active alert events should be resent."
+                name="Periodic Alert Interval (minutes)" id="period" required="true"
+                type="Integer" default="1440">
+        </AD>
+        <AD description="The minimum alert priority that should be sent to the periodic topic. 4=Critical 3=Important 2=Normal 1=Low"
```

Comment:
❓Should we provide a simple example for this one, e.g., `for instance, if the priority is set to 2, only Normal priority alerts and above will be sent`?